pkg/diff/diff.go

```diff
@@ -208,6 +208,10 @@ func removeWebhookMutation(predictedLive, live *unstructured.Unstructured, gvkPa
 	}
 	gvk := predictedLive.GetObjectKind().GroupVersionKind()
 	pt := gvkParser.Type(gvk)
+	if pt == nil {
+		return nil, fmt.Errorf("unable to resolve parseableType for gvk: %s", gvk)
```

Comment:
```suggestion
		return nil, fmt.Errorf("unable to resolve parseableType for gvk (Group Version Kind): %s", gvk)
```
I think we can make this a bit more clear.

Reply:
@todaywasawesome This is an internal error message but I agree with making it a bit more clear. I added a small modification. 
PTAL